src/main/scala/chisel3/util/HasDemoBundle.scala

```diff
@@ -0,0 +1,426 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+//package chisel3
+//
+//import scala.language.reflectiveCalls
+//
+//import chisel3.experimental.{ChiselEnum, FixedPoint}
+//import chisel3.stage.ChiselStage
+//import chisel3.util.Decoupled
+//
```

Comment:
I'm assuming you didn't mean to include this file?

Reply:
Sheepishly did mean to leave it in there for the moment, I guess I could tuck it somewhere else, but in case there are problems I uncomment specific blocks and am able to edit compile debug much quicker. Happy to remove it though

Reply:
We can keep it for now but delete before merging, no worries

Reply:
Will Delete in next commit